client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.ts

```diff
@@ -42,6 +42,34 @@ export const siteColumns = [
 	},
 ];
 
+export const siteColumnswithStats = [
+	{
+		key: 'site',
+		title: translate( 'Site' ),
+	},
+	{
+		key: 'stats',
+		title: translate( 'Stats' ),
+	},
+	{
+		key: 'backup',
+		title: translate( 'Backup' ),
+	},
+	{
+		key: 'scan',
+		title: translate( 'Scan' ),
+	},
+	{
+		key: 'monitor',
+		title: translate( 'Monitor' ),
+		className: 'min-width-100px',
+	},
+	{
+		key: 'plugin',
+		title: translate( 'Plugin Updates' ),
+	},
+];
```

Comment:
Is there a better way to do this, without having to duplicate the variable?

Reply:
We could do something like

```
const isExpandedBlockEnabled = config.isEnabled( 'jetpack/pro-dashboard-expandable-block' );

const extraColumns = isExpandedBlockEnabled
	? [
			{
				key: 'stats',
				title: translate( 'Stats' ),
			},
	  ]
	: [];

export const siteColumns = [
	{
		key: 'site',
		title: translate( 'Site' ),
	},
	extraColumns,
	{
		key: 'backup',
		title: translate( 'Backup' ),
	},
	{
		key: 'scan',
		title: translate( 'Scan' ),
	},
	{
		key: 'monitor',
		title: translate( 'Monitor' ),
		className: 'min-width-100px',
	},
	{
		key: 'plugin',
		title: translate( 'Plugins' ),
	},
];

```

Reply:
Great idea, thanks!